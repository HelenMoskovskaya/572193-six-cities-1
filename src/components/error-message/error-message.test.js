import React from 'react';
import renderer from 'react-test-renderer';
import {MemoryRouter} from 'react-router';
import ErrorMessage from './error-message.jsx';


describe(`ErrorMessage`, () => {
  it(`renders component correctly`, () => {
    const tree = renderer.
      create(<MemoryRouter>
        <ErrorMessage
        />
      </MemoryRouter>).
      toJSON();

    expect(tree).toMatchSnapshot();
  });
});
