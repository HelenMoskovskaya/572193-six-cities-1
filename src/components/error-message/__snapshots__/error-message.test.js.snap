// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ErrorMessage renders component correctly 1`] = `
<div
  className="page page--gray page--main"
>
  <div
    style={
      Object {
        "color": "#4481C3",
        "fontSize": "35px",
        "textAlign": "center",
      }
    }
  >
    <p>
      Sorry, our site is temporarily unavailable.
    </p>
  </div>
</div>
`;
